### ISSUE TITLE
Encouraging contributions

### ISSUE THREAD

**Issue by lukedeo** [farizrahman4u/keras-contrib#9]
I feel like we should do a PR push (i.e., Twitter) to get people contributing and interested here. Thoughts?

**Comment by the-moliver** [comment]
I agree, but I think @fchollet would get the most traction with a tweet

**Comment by ahundt** [comment]
I assume keras itself has a contributions document, I suggest adding a reference to this there as well, explaining the process of first contributions going here then how eventual promotion to mainline keras would work.

**Comment by kemaswill** [comment]
By the way, I think we'd better open an issue for contribution request to list some new functions that we need, such as a list of new activations, layers, optimizers and examples, just like https://github.com/fchollet/keras/issues/3617. I'll do a quick research review to give an initial list, any suggestions will be welcomed.

**Comment by jpeg729** [comment]
I think the best way to encourage contributions would be to have a website to showcase the contributions.

**Comment by Dref360** [comment]
Maybe a github.page where every contirbutors can showcase their add? It's super easy to setup and it's free

**Comment by bstriner** [comment]
Hey, guys! Finally should have free time over the summer.

Every once in a while a good idea pops up on the forums and I try to refer it over here. The hardest part is just getting the ball rolling.

`keras-contrib` is referenced in the keras `contributing.md` but mayber there are more prominent locations that might be appropriate, such as the issue or PR submission forms.

A few repos I might want to clean up and consolidate into keras-contrib:
* https://github.com/bstriner/keras-adversarial Ability to combine multiple keras models into a single keras model so you can do multiplayer optimization, such as GANs.
* https://github.com/bstriner/dense_tensor Neural Tensor Network. Normal dense layers are `y=a(xW+b)`. Neural Tensor Layers are `y=a(xUx+xW+b)`, so it is like a quadratic extension. Really simple but cool stuff and not deep enough research on this out there.
* https://github.com/bstriner/keras-tqdm Link between keras and TQDM. Makes training keras models look really nice in IPython but you can use it on the console, too. Would probably need to make it an extension of the install so it could handle the additional dependency.
* https://github.com/bstriner/bayesian_dense  Bayesian weight uncertainty. Unique idea for regularization. Layer learns a gaussian model of weights. Weights are samples from that model and used in the network and backprop works through the reparameterization trick.

Cheers

**Comment by ahundt** [comment]
sounds cool, there are also a bunch of items in the set of issues: 
https://github.com/farizrahman4u/keras-contrib/issues

**Comment by bstriner** [comment]
A few good ones in the keras issues, too. The idea of better generators for image data and text data would be really useful.
